geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/index/GeoMesaSchemaValidator.scala

```diff
@@ -61,6 +82,7 @@ object TemporalIndexCheck extends LazyLogging {
 object MixedGeometryCheck extends LazyLogging {
 
   import java.lang.{Boolean => jBoolean}
+
```

Comment:
More interesting than that, I think:  I organized imports here, and that likes to separate java.* imports from other packages.  If you like, I can remove the blank line.